cpuid/src/common.rs

```diff
@@ -16,32 +16,28 @@ use std::arch::x86::__cpuid_count;
 #[cfg(target_arch = "x86_64")]
```

Comment:
Looks like this commit (ba3d2258dfa51c63afa6649aac48e469f4466334) is just moving the code from the previous commit. I understand that the previous commit was already reviewed as part of the #882 PR, but other than that is there any reason for not squashing these two commits together?